koku/masu/test/celery/test_tasks.py

```diff
@@ -345,9 +345,7 @@ def setUp(self):
         """Set up initial data for tests."""
         super().setUp()
 
-        with ReportingCommonDBAccessor(self.schema) as common_accessor:
-            self.column_map = common_accessor.column_map
-        self.creator = ReportObjectCreator(self.schema, self.column_map)
+        self.creator = ReportObjectCreator(self.schema, REPORT_COLUMN_MAP)
```

Comment:
It looks like this is still in the constructor too.

Reply:
updated @kholdaway